server/src/main/java/org/opensearch/cluster/service/TaskBatcher.java

```diff
@@ -195,25 +195,37 @@ void runIfNotProcessed(BatchedTask updateTask) {
             if (toExecute.isEmpty() == false) {
                 Function<Boolean, String> taskSummaryGenerator = (longSummaryRequired) -> {
                     if (longSummaryRequired == null || !longSummaryRequired) {
-                        return buildShortSummary(updateTask.batchingKey, toExecute.size());
+                        final List<BatchedTask> sampleTasks = toExecute.stream()
+                            .limit(Math.min(1000, toExecute.size()))
```

Comment:
let keep this as 100

Reply:
ack

Reply:
@Bukhtawar Will 100 tasks summary work for node-join and node-left use-cases?

Reply:
We will keep it to 1000 for now.